cms/templates/structured_tags_block.html

```diff
@@ -1,16 +1,16 @@
-<div class="xblock-render">
+<div class="xblock-render" class="studio-xblock-wrapper">
     % for tag in tags:
-    <label for="problem_tags_${tag['key']}">${tag['title']}</label>:
-    <select id="problem_tags_${tag['key']}" name="${tag['key']}">
+    <label for="tags_${tag['key']}">${tag['title']}</label>:
+    <select id="tags_${tag['key']}" name="${tag['key']}">
```

Comment:
@cpennington you are right that there could be situation with many `structured_tags_blocks` with the same `id`. It does not affect functionality because these `id` attribute doesn't used from the connected JavaScript but from the HTML point of view it is really looks doubtfully so I've changed this behaviour and added `block.location` id to make this value unique